packages/gx-agent-cli/lib/participant.ts

```diff
@@ -47,7 +47,7 @@ sd.command('submit')
         }
         selfDescription = await agent.acquireComplianceCredentialFromUnsignedParticipant({
           credential: sd,
-          persist: true,
+          persist: cmd.persist,
```

Comment:
I don't want this optional. Alternatively it could be made to check for a false value, but if undefined or true it should really store the VPs in the agent

Reply:
reverted